src/components/Layout.js

```diff
@@ -7,8 +7,9 @@ import Sidebar from "./Sidebar/Sidebar";
 const Layout = () => {
   return (
     <div className={styles.Layout}>
-      <Header />
-      <Sidebar />
+      <Sidebar>
+        <Header />
```

Comment:
Do you really mean `Dashboard` inside `Sidebar`? Shouldn't it be rather a separate component?

Reply:
all of them are separete components, in this case Header component is inside Sidebar (child of Sidebar) not Dashboard ;)